README.prod.md

```diff
@@ -284,6 +284,24 @@ using topics:
 * You can edit topics to fix a thread that's already been started,
   which can be helpful when onboarding new batches of users to the platform.
 
+Third, setting default streams for new users is a great way to get
+new users involved in conversations before they've accustomed
+themselves with joining streams on their own. You can use the
+[`set_default_streams`](https://github.com/zulip/zulip/blob/master/zerver/management/commands/set_default_streams.py) command to set default streams for users within a realm:
+
+```
+python manage.py set_default_streams --domain=zulip.com --streams=foo,bar,...
```

Comment:
maybe do `--domain=example.com` here to avoid confusion?  There is a "zulip.com" domain that gets auto-created for certain bots that aren't part of any realm to live in (which we probably will want to rename at some point...).